src/test/java/gov/cabinetofice/gapuserservice/web/LoginControllerV2Test.java

```diff
@@ -385,6 +390,8 @@ void shouldRequireReauthentication_ifNonceIsExpired() throws JSONException {
             final HttpServletResponse response = Mockito.spy(new MockHttpServletResponse());
             final JSONObject tokenResponse = new JSONObject();
             tokenResponse.put("id_token", idToken);
+            tokenResponse.put("tokenHint", "blah");
+            tokenResponse.put("access_token", "blah");
```

Comment:
access_token value should probably be the same in these 3 tests as they're unrelated to this change